src/main/kotlin/io.javalin/ApiBuilder.kt

```diff
@@ -0,0 +1,133 @@
+/*
+ * Javalin - https://javalin.io
+ * Copyright 2017 David Åse
+ * Licensed under Apache 2.0: https://github.com/tipsy/javalin/blob/master/LICENSE
+ */
+
+package io.javalin
+
+import io.javalin.security.Role
+import java.util.*
+
+object ApiBuilder {
+
+    private var staticJavalin: Javalin? = null
+    private val pathDeque = ArrayDeque<String>()
+
+    @FunctionalInterface
+    interface EndpointGroup {
+        fun addEndpoints()
+    }
+
+    @JvmStatic fun setStaticJavalin(javalin: Javalin) {
+        staticJavalin = javalin
+    }
+
+    @JvmStatic fun clearStaticJavalin() {
+        staticJavalin = null
+    }
+
+    @JvmStatic fun path(path: String, endpointGroup: EndpointGroup) {
+        pathDeque.addLast(path)
+        endpointGroup.addEndpoints()
+        pathDeque.removeLast()
+    }
+
+    @JvmStatic private fun prefixPath(path: String): String {
+        return pathDeque.joinToString("") + path
+    }
+
+    // Everything below here is copied from the end of Javalin.java
```

Comment:
I think so. I need to inform whoever is looking at it that the methods are just static aliases.